docs/configuration/sample-config.md

```diff
@@ -2,39 +2,100 @@
 id: sample-config
 title: Server Config File
 ---
+# Server configuration file
+Starting the server with various configuration options can be done using a configuration file instead of using just flags.  
+The command used to start the server with config file `polygon-edge server --config <config_file_name>`
```

Comment:
Typo here, should be `with a config file`

Reply:
Fixed 1713f6a24cb0da0f003c85013685ea41d8b92152